packages/expo-document-picker/android/src/main/java/expo/modules/documentpicker/DocumentPickerModule.kt

```diff
@@ -132,6 +132,6 @@ class DocumentPickerModule(
       return null
     }
 
-    return outputFilePath
+    return "file://$outputFilePath"
```

Comment:
```suggestion
    return "file://$outputFilePath"
```
Are we sure if the `outputFilePath` won't contain a file scheme even if someone using a custom document picker? 
I think no. We need to wrap it in something which will be a little more intelligent. Maybe something like:
https://github.com/expo/expo/blob/22b3ce370fa3848e1046177201e5daa57ca2bc09/packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt#L36